src/auth-service/routes/v2/users.js

```diff
@@ -1049,8 +1049,8 @@ router.get(
 );
 
 /*********************************** user notifications **********************/
-router.post(
-  "/subscribe/:type",
+router.get(
+  "/subscribe/:product/:type",
```

Comment:
Hi @BenjaminSsempala , thanks for this.

- Just curious, why are we still adding` product` as a query parameter in our endpoints?
- Considering the `one account` direction, isn't `type` just enough?

More than happy to dig further into this in a sync of sorts. Thanks again.